presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerClient.java

```diff
@@ -428,8 +428,9 @@ public void setNull(PreparedStatement statement, int index)
         return handle.createQuery("" +
                 "SELECT data_compression_desc FROM sys.partitions p " +
                 "INNER JOIN sys.tables t ON p.object_id = t.object_id " +
+                "INNER JOIN sys.schemas s ON t.schema_id = s.schema_id " +
                 "INNER JOIN sys.indexes i ON t.object_id = i.object_id " +
-                "WHERE SCHEMA_NAME(t.schema_id) = :schema AND t.name = :table_name AND i.type IN (0,1) " +
```

Comment:
replacing SCHEMA_NAME with INNER JOIN sys.schemas looks like independent change.
please split this into 2 commits?